src/Util/GameObject.cpp

```diff
@@ -7,7 +8,13 @@ void GameObject::Draw() {
         return;
     }
 
-    m_Drawable->Draw(m_Transform, m_ZIndex);
+    Util::Transform offsetTransform {
+        m_Transform.translation - m_Pivot,
+        m_Transform.rotation,
+        m_Transform.scale,
+    };
+
+    m_Drawable->Draw(offsetTransform, m_ZIndex);
```

Comment:
I'm not sure what this code can do. It seems like `m_Pivot` and `m_Transform.translation` serve the same function.